lists/ae/ae-aj-cr.json

```diff
@@ -12,7 +12,7 @@
     "coverage": [
         "AE"
     ],
-    "code": "AE-AJ-CR",
+    "code": "AE-AJ_CR",
```

Comment:
@timgdavies [the handbook](http://docs.org-id.guide/en/latest/metadata/#list-name-code) states that underscores are used for multilingual codes. However, here the part before the underscore is a subnational division code. Is that correct?